src/frontend/components/DowloadVideo/DownloadVideo.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from 'grommet';
+import * as React from 'react';
+import { defineMessages, FormattedMessage, Messages } from 'react-intl';
+
+import { Video, videoSize } from '../../types/tracks';
+
+const messages: Messages = defineMessages({
+  downloadVideo: {
+    defaultMessage: 'Download video with quality: ',
+    description: 'Text to download the video',
+    id: 'components.DownloadVideo.downloadVideo',
+  },
+  480: {
+    defaultMessage: 'Mobile',
+    description: 'Mobile quality video',
+    id: 'components.DownloadVideo.480',
+  },
+  720: {
+    defaultMessage: 'Normal',
+    description: 'Normal quality video',
+    id: 'components.DownloadVideo.720',
+  },
+  1080: {
+    defaultMessage: 'Hight',
+    description: 'Hight quality video',
+    id: 'components.DownloadVideo.1080',
+  },
+});
+
+export const DownloadVideo = ({ video }: { video: Video }) => {
+  const elements: JSX.Element[] = [];
+  return (
+    <Box align="center" justify="start" direction="row" pad="small">
+      <FormattedMessage {...messages.downloadVideo} />
+      &nbsp;
+      {(['1080', '720', '480'] as videoSize[])
+        .reduce((acc: JSX.Element[], size: videoSize) => {
+          acc.push(
+            <a
+              key={size}
+              target="_blank"
+              rel="noopener noreferrer"
+              href={video.urls.mp4[size]}
+            >
+              <FormattedMessage {...messages[size]} /> ({size}p)
```

Comment:
I think you should only write the size here and add a title to the link with detailed info. I've tried this:

![peek 2019-03-06 17-03](https://user-images.githubusercontent.com/956157/53895246-de613b00-4031-11e9-8579-752321f75bd2.gif)

Reply:
But this is not mobile friendy :thinking:

Reply:
Nope. But IMO it's already rather explicit without the titles and the interface is cleaner.

Reply:
You suppose that everybody know what is a resolution. I will ask to madame michu if she knows what is a `1080p`.

Reply:
Maybe can we had an "info" (ℹ) icon on the right side explaining

* 1080p: FullHD (high quality)
* 720p: HD (standard)
* 480p: SD (Mobile)

WDYT?

Reply:
That's better. I will see what I can do :wink:

Reply:
![peek 2019-03-07 10-57](https://user-images.githubusercontent.com/767834/53948328-d8b63480-40c7-11e9-8379-63dfbdfc2d3f.gif)

What do you think about this ?

Reply:
you should add a cursor:pointer; on the `i`

Reply:
Can we click on the info icon to display a tooltip instead of an hover event? Also I would change the message: "Download video with quality" by something like "Video downloads:". WDYT?

Reply:
Done